buildsystem/python.cmake

```diff
@@ -137,7 +138,8 @@ function(add_cython_modules)
 				set_property(GLOBAL APPEND PROPERTY SFT_CYTHON_MODULES_EMBED "${source}")
 				add_executable("${TARGETNAME}" "${CPPNAME}")
 
-				target_link_libraries("${TARGETNAME}" "${PYEXT_PYLIB}")
+				# TODO: use full ldflags and cflags provided by python${VERSION}-config
+				target_link_libraries("${TARGETNAME}" "-L${PYEXT_LIBRARY_DIR} -l${PYEXT_LIBRARY_NAME}")
```

Comment:
In the other modules, the variables have names like

    FTGL_LIBRARIES
    BACKTRACE_LIBRARY
    OPUS_LIBRARIES
    SDL2_LIBRARIES

Even though they clearly contain linker flags, not (file)names. I agree that it doesn't really make sense, so... choose however you wish, but wisely.